gradle-plugin/src/test/resources/javaOutput/land-micronaut/OrtResource.java.txt

```diff
@@ -40,6 +42,7 @@ public class OrtResource {
     return OPTIONS_CONTENT;
   }
 
+  @Status(HttpStatus.NO_CONTENT)
```

Comment:
The resource returns a `HttpReponse`. The status can be set in there.

Thing is that this implementation is confusing because

1. I as a developer can set the status code in `HttpResponse` with my resource delegate
2. The resource itself declares `@Status`

I'm not sure what Micronaut now respects. We need to investigate

Reply:
Changed return type to `Void`